buildifier/integration_test.sh

```diff
@@ -15,11 +15,13 @@ INPUT="load(':foo.bzl', 'foo'); foo(tags=['b', 'a'],srcs=['d', 'c'])"  # formatt
 echo -e "$INPUT" > test/build  # case doesn't matter
 echo -e "$INPUT" > test/test.bzl
 echo -e "$INPUT" > test/subdir/test.bzl
+echo -e "$INPUT" > test.bzl  # outside the test directory
 echo -e "not valid +" > test/foo.bar
 cp test/foo.bar golden/foo.bar
 
 "$buildifier" < test/build > stdout
-"$buildifier" test
+"$buildifier" -r test
```

Comment:
Do you have a test for `$buildifier" test`?

Reply:
Added a test

Reply:
The test fails because the error message is "is a directory" on Linux/Mac and "The handle is invalid." on Windows. Is there a simple workaround (e.g. a way to specify different test strings on different platforms)?

Reply:
The test is not so important (matching against the system provided string is brittle).
Maybe just check the error code.